src/tokens/selectors.ts

```diff
@@ -432,38 +432,20 @@ export const spendTokensByNetworkIdSelector = createSelector(
   (tokens) => tokens.filter((tokenInfo) => networkConfig.spendTokenIds.includes(tokenInfo.tokenId))
 )
 
-const tokensWithBalanceOrShowZeroBalanceSelector = createSelector(
-  (state: RootState, networkIds: NetworkId[]) => tokensListSelector(state, networkIds),
-  (tokens) =>
-    tokens.filter(
-      (tokenInfo) => tokenInfo.balance.gt(TOKEN_MIN_AMOUNT) || tokenInfo.showZeroBalance
-    )
-)
+// const tokensWithBalanceOrShowZeroBalanceSelector = createSelector(
+//   (state: RootState, networkIds: NetworkId[]) => tokensListSelector(state, networkIds),
+//   (tokens) =>
+//     tokens.filter(
+//       (tokenInfo) => tokenInfo.balance.gt(TOKEN_MIN_AMOUNT) || tokenInfo.showZeroBalance
+//     )
+// )
 
 export const sortedTokensWithBalanceOrShowZeroBalanceSelector = createSelector(
-  tokensWithBalanceOrShowZeroBalanceSelector,
+  (state: RootState, networkIds: NetworkId[]) => tokensListSelector(state, networkIds),
```

Comment:
get all of the tokens (since Jacob's changes make it so only cKES and cUSD can be returned